docs/topics/native/native-binary-licenses.md

```diff
@@ -0,0 +1,73 @@
+   </tr>
+   <tr>
+        <td><a href="https://www.gwtproject.org/">GWT</a></td>
+   </tr>
+   <tr>
+        <td><a href="https://guava.dev">Guava</a></td>
+   </tr>
+   <tr>
+        <td><a href="https://github.com/ianlancetaylor/libbacktrace">libbacktrace</a></td>
+        <td><a href="https://github.com/JetBrains/kotlin/blob/master/kotlin-native/licenses/third_party/libbacktrace_LICENSE.txt">3-clause BSD license with copyright notice</a></td>
+   </tr>
+   <tr>
+        <td><a href="https://github.com/microsoft/mimalloc">mimalloc</a></td>
+        <td>
+          <p><a href="https://github.com/JetBrains/kotlin/blob/master/kotlin-native/licenses/third_party/mimalloc_LICENSE.txt">MIT license</a></p>
+          <p>Always include, unless you use the system memory allocator (the <code>-Xallocator=std</code> compiler option is set). For more information on allocators, see <a xref="native-memory-manager.md">Kotlin/Native memory management</a></p>
+        </td>
+   </tr>
+   <tr>
+        <td><a href="https://www.unicode.org/">Unicode character database</a></td>
+        <td><a href="https://github.com/JetBrains/kotlin/blob/master/kotlin-native/licenses/third_party/unicode_LICENSE.txt">Unicode license</a></td>
+   </tr>
+</table>
+
+Specific targets additionally require the following license files:
+
+| Project                                                               | Targets  | Files to be included                                                                                                                                                                                                                                                                                                              | 
+|-----------------------------------------------------------------------|----------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [MinGW-w64 headers and runtime libraries](https://www.mingw-w64.org/) | `mingw*` | <list><li><a href="https://sourceforge.net/p/mingw-w64/mingw-w64/ci/master/tree/COPYING.MinGW-w64-runtime/COPYING.MinGW-w64-runtime.txt">MinGW-w64 runtime license</a></li><li><a href="https://sourceforge.net/p/mingw-w64/mingw-w64/ci/master/tree/mingw-w64-libraries/winpthreads/COPYING">Winpthreads license</a></li></list> |
+| [Musl (math implementation)](https://musl.libc.org/)                  | `wasm32` | [Musl copyright notice](https://github.com/JetBrains/kotlin/blob/master/kotlin-native/runtime/src/main/cpp/math/COPYRIGHT)                                                                                                                                                                                                        |
```

Comment:
Shouldn't we delete this because it's an old version of Kotlin/Wasm?

Reply:
I asked about it, the team says we still need to keep it as of now, the old wasm32 target is deprecated since 1.8.20, but not removed yet